research-software-directory/RSD-as-a-service
# 807 organisation card

# Update organisation card with additional info

Closes #807

Changes proposed in this pull request:
* add organisation country and short description to organisation card
* consolidate card components between organisation and project
* update organisation unit test

How to test:
* `make start` to build app and generate test data
* navigate to organisation overview
* confirm card contain country and short description

## Example organisation overview (cards)

![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/4b5f074b-ec61-4eed-a06a-2c7320aac419)

PR Checklist:
